components/navbar/navbar.jsx

```diff
@@ -210,7 +210,7 @@ class NavBar extends React.Component {
     // For more info see: https://github.com/mozilla/network-pulse/issues/326
 
     return (
-      <div className="navbar mb-5">
```

Comment:
Note: This was removed to for more flexibility with individual components/pages e.g. https://redpen.io/xpaad30985723f1669.